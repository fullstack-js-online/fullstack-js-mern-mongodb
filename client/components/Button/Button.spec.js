import React from 'react'
import { mount } from 'enzyme'

// component
import Button from './index'

describe('The button component', () => {
  const props = {
    loading: true
  }

  it('should match snapshot', () => {
    const button = mount(<Button {...props}>Sign up</Button>)

    expect(button).toMatchSnapshot()
  })
})
