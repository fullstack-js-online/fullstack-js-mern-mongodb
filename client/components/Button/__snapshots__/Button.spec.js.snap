// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The button component should match snapshot 1`] = `
<Button
  loading={true}
>
  <button
    className="focus:outline-none rounded-sm hover:bg-emerald-light px-4 py-5 w-full text-white text-xl bg-emerald"
    disabled={true}
    type="button"
  >
    <Loader
      dark={false}
    >
      <img
        className="loader"
        src="/loading.png"
      />
    </Loader>
  </button>
</Button>
`;
